specifyweb/frontend/js_src/lib/components/Attachments/Gallery.tsx

```diff
@@ -74,7 +74,7 @@ export function AttachmentGallery({
     <>
       <Container.Base
         className="grid flex-1 grid-cols-[repeat(auto-fit,minmax(var(--scale),1fr))]
-          items-center gap-4"
```

Comment:
this shadow should be there when rendered as a standalone page. only remove shadow when in a dialog.

when making changes please be careful to look at all the usages of a component to check the reputation of a change

Reply:
Dismissing this as the change was intentional